modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -146,7 +148,9 @@ async fn sanity_check_bitcoin_blocks() -> anyhow::Result<()> {
 #[tokio::test(flavor = "multi_thread")]
 async fn on_chain_peg_in_and_peg_out_happy_case() -> anyhow::Result<()> {
     let fixtures = fixtures();
-    let fed = fixtures.new_fed().await;
+    let span = tracing::info_span!("on_chain_peg_in_and_peg_out_happy_case");
```

Comment:
Why create a span, but then enter if after `new_fed`?

Reply:
:thinking:  You mean, why not enter it before? I guess I could do that. Perhaps add `#[instrument]` then create a span just for `new_fed`